src/ngraph/op/op_version_tbl.hpp

```diff
@@ -164,7 +164,7 @@ NGRAPH_OP(PRelu, ngraph::op::v0, 0)
 NGRAPH_OP(PSROIPooling, ngraph::op::v0, 0)
 NGRAPH_OP(Pad, ngraph::op::v0, 0)
 NGRAPH_OP(Pad, ngraph::op::v1, 1)
-NGRAPH_OP(Parameter, ngraph::op::v0, 0)
+NGRAPH_OP(Parameter, ngraph::op, 0)
```

Comment:
This looks like a merge artifact. Please fix it, if you didn't mean to add this change.

Reply:
Done